BDArmory/Modules/MissileFire.cs

```diff
@@ -1163,7 +1170,7 @@ void UpdateCurrentHP(Vessel v)
             { currentHP = GetTotalHP(); }
         }
 
-        public override void OnUpdate()
+        public override void OnUpdate() //shouldn't most of this stuff be in onFixedUpdate instead?
```

Comment:
There's a bunch of missile targeting stuff that happens here; how does this play with new BDA time accel tools?

Reply:
Yeah, most likely. The stuff hidden behind `if (vessel.isActiveVessel)` is for mouse control, so that's fine in `OnUpdate`, but the part before it should probably be in `OnFixedUpdate`.

The time acceleration mostly works fine for using guns (some small differences in the results, but I'm not sure how statistically significant they are) and some missile types seem to work OK, but other missile types (like the AIM-9) don't work properly at all.